### ISSUE TITLE
Focus order (WCAG 2.4.3)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1237]
**Request:** The intent of this Success Criterion is to ensure that when users navigate sequentially through content, they encounter information in an order that is consistent with the meaning of the content and can be operated from the keyboard.

**Issues:** 
1. https://slidewiki.org/presentation/81127-1/overview-of-accessibility/811271/437649-3/?language=en-/slide-719796-3
-In the case of keyboard navigation, invisible areas are activated which make navigation with the keyboard considerably more difficult. Only the important parts of the page (e.g. menu, arrows) should be able to be reached easily with the keyboard.

2. https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/edit 
- After closing the language selection drop-down menu, the focus returns to the beginning of the page and does not remain with the selected menu. After closing the menu, the focus should remain in the appropriate place.

![grafik](https://user-images.githubusercontent.com/48785136/74322975-11a16380-4d85-11ea-8e70-f4ba346c2e80.png)

**Comment by aoelen** [comment]
1. This issue will be fixed for newly created slides. Originally, a tab-index was added for all elements within a slide. This resulted in non-interactive elements to be focusable. This is fixed now, but unfortunately, it is not easily possible to update existing slides 

2. For me (in Chrome), the focus remains on the dropdown after closing. Can you confirm if this is still an issue.

**Comment by VivienDZB** [comment]
It is still an issue in Firefox. The menu is closable with Esc but the focus still returnes to the beginning of the page.

**Comment by aoelen** [comment]
I tried to reproduce this in Firefox 81.0 and it seems to work fine. What I did:

* Navigate with the keyboard to the language selector 
* Use the arrow keys up and down
* Then closing the menu with the Esc key 
* When I use the tab key again, the focus goes to the "Open slideshow in new tab" button (as expected)

Can you check if this issue is solved now?

**Comment by VivienDZB** [comment]
Works just fine. :)

**Comment by aoelen** [comment]
Happy to hear that, thanks for checking!
